corda/corda-runtime-os
# CORE-14719 - New version of config schema caused failure on platform upgrade

Config schema changes were not picked up upon platform upgrade. We will use a default value until a forced upgrade happens via the config reconciler which results in values being received to configure the ExpirationProcessor.

Testing:
1. Run a deployment in 5.0.
2. Do platform upgrade to 5.1.
3. Check the logs: member worker runs okay and processor could start:
![log1](https://github.com/corda/corda-runtime-os/assets/61757742/0d858d3e-2b45-458d-bb50-3a87986c1567)
4. Check the current config in DB (it should have old values at this point):
![lookup1](https://github.com/corda/corda-runtime-os/assets/61757742/6dcf74af-2f9a-482e-8840-e56da66b8cb8)
5. Update the config with empty value to force the reconciler to run:
![update](https://github.com/corda/corda-runtime-os/assets/61757742/dd026af9-fdcc-47d4-ac99-4f6c7cee3d08)
6. Check the current config in DB (it should have the new values at this point as defined in the api project):
![lookup2](https://github.com/corda/corda-runtime-os/assets/61757742/79035a69-dddd-4767-8a4d-545a8fa3beae)
7. Check the logs to see that ExpirationProcessor receive the config and restarted:
![log2](https://github.com/corda/corda-runtime-os/assets/61757742/80e1bc95-3e77-4117-968d-1992ea0237d8)
